lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
```

Comment:
@peterdolan the reason we have tags is because they are used for our docs plugin but have no other use case (other than context as you're suggesting). I would not add things if they are not actually used.